src/app/components/PopUpBox/PopUpBox.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Typography from '../Typography/Typography';
+import Button from '../Button/Button';
+import type { ReactNode } from 'react';
+import style from './PopUpBox.module.css';
+
+export type PopUpBoxProps = {
+  children: ReactNode;
+  boxType: 'infoBox' | 'statusBox';
+  className?: string;
+  onClose: () => void;
+  changeKeimen: () => void;
+  changeEinweichen: () => void;
+};
+
+const PopUpBox = ({
+  children,
+  className,
+  boxType,
+  onClose,
+  changeKeimen,
+  changeEinweichen,
+}: PopUpBoxProps): JSX.Element => {
+  const BoxMap = {
+    infoBox: (
+      <div className={style.infobox}>
+        <Typography size="s">{children}</Typography>
+        <Button
+          className={style.infoboxButton}
+          onClick={onClose}
+          children="OK"
+        />
+      </div>
+    ),
+    statusBox: (
+      <div className={style.statusbox}>
+        <Typography size="s">{children}</Typography>
+        <Button onClick={changeEinweichen} children="Einweichen" />
+        <Button onClick={changeKeimen} children="Keimen" />
+      </div>
+    ),
+  };
+  return (
+    <article className={`${style.box} ${className}`}>{BoxMap[boxType]}</article>
```

Comment:
This would be even more awesome if it had a backdrop that spans the entire page, with a blur filter or some transparent color, and when you click it, it triggers the same `onClose` function as the close button. So a parent `div` element for this `article`. Just a suggestion, not crucial of course

Reply:
Thank you for your suggestion. I will do this later and open an extra issue.